askbot/media/style/style.css

```diff
@@ -5004,9 +5004,6 @@ td.setting-input textarea {
   .short-summary .tags li {
     margin: 8px 8px 1px 0;
   }
-  #secondaryHeader {
```

Comment:
Now that there's only one line of content, we don't need to override the header height on mobile